servers/quarkus-server/src/test/java/org/projectnessie/server/ITNativeRest.java

```diff
@@ -33,15 +28,7 @@
   @Override
   @BeforeEach
   public void setUp() throws Exception {
-    URI uri = URI.create("http://localhost:19121/api/v1");
```

Comment:
we need to init the NessieApiV1 here and just pass it down for grpc

Reply:
See https://github.com/projectnessie/nessie/pull/2011#discussion_r711102482